tron/api/resource.py

```diff
@@ -424,6 +424,41 @@ def render_GET(self, request):
         return respond(request, view_all_metrics())
 
 
+class EventsResource(resource.Resource):
+    isLeaf = True
+
+    def __init__(self):
+        super().__init__()
+        self.controller = controller.EventsController()
+
+    @AsyncResource.exclusive
```

Comment:
Is this right? Only one client can read the event log at a time, while multiple clients can publish/discard?